.vscode/launch.json

```diff
@@ -4,12 +4,20 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
-        {
-            "name": "Attach to Process",
-            "type": "go",
-            "request": "attach",
-            "mode": "local",
-            "processId": 31790
-        }
+      {
+        "name": "Launch test function",
+        "type": "go",
+        "request": "launch",
+        "mode": "test",
+        "program": "${workspaceFolder}/pkg/convert",
+        "args": ["-test.run", "TestTranslate/template_join_expr"]
```

Comment:
Accidental add?

Reply:
I thought it might be ok but yeah it is kind of specific